x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -157,11 +158,12 @@
      * Cheat Sheet</a> and the <a href="https://pages.nist.gov/800-63-3/sp800-63b.html#sec5">
      * NIST Digital Identity Guidelines</a>
      */
-    private static final int ITERATIONS = 100000;
+    static final int TOKEN_SERVICE_KEY_ITERATIONS = 100000;
+    static final int TOKENS_ENCRYPTION_KEY_ITERATIONS = 1024;
```

Comment:
The key is derived from a random sting, we don't need too many iterations, we want this to be quick and not computationally expensive.